manifests/logrotate.pp

```diff
@@ -1,30 +1,26 @@
-# == Class aide::logrotate
-#
 # A class that sets up the logrotate state for aide.
 #
-# == Parameters
+# @param logdir Directory containing the logs to be rotated.
+#   The logs in that directory are assumed to end with '.log'.
 #
-# [*rotate_period*]
+# @param rotate_period
 #   The logrotate period at which to rotate the logs.
 #
-# [*rotate_number*]
+# @param rotate_number
 #   The number of log files to preserve on the system.
 #
-# == Authors
-#
-# * Trevor Vaughan <tvaughan@onyxpoint.com>
+# @author https://github.com/simp/pupmod-simp-aide/graphs/contributors
 #
 class aide::logrotate (
+  Stdlib::Absolutepath    $logdir        = $::aide::logdir,
   Aide::Rotateperiod      $rotate_period = 'weekly',
```

Comment:
Do you want to make these parameters and the parameters in the cron class available at the top level, through init.pp? Then this module would be completely configurable.